charts/test/ingress_test.go

```diff
@@ -0,0 +1,52 @@
+package test
+
+import (
+	"io/ioutil"
+	"reflect"
+	"testing"
+
+	"gopkg.in/yaml.v2"
+
+)
+
+func Test_IngressAuthNoTLS(t *testing.T) {
+	parts := []string{
+		"--set", "edgeAuth.clientSecret=200",
+		"--set", "edgeAuth.clientId=20",
+		"--set", "global.customersURL=20",
+		"--set", "global.rootDomain=20",
+	}
+
+	annotations := make(map[string]string)
+	annotations["kubernetes.io/ingress/class"] = "nginx"
+	annotations["nginx.ingress.kubernetes.io/limit-connections"] = "20"
+	annotations["nginx.ingress.kubernetes.io/limit-rpm"] = "600"
+
+
+	want := ingressAuth{
+		ApiVersion: "appa/v1",
+		Kind:       "Ingress",
+		Metadata:   MetadataItems{
+			Name:        "openfaas-auth-ingress",
+			Annotations: annotations,
+		},
+		Spec:       Spec{},
+	}
+	_, _ = helmRunner(parts...)
```

Comment:
Doesn't this return anything that could be checked? The output path for instance would be better returned than hard-coded if possible. Can it throw an error like helm not found?